### PR TITLE
dev-db/mongodb: bump to 3.6.7

### PR DESCRIPTION
**Pull request by hydrapolic**
Tests pass on ~amd64. Thanks :)

### CODE REVIEW

File: dev-db/mongodb/mongodb-3.6.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+SCONS_MIN_VERSION="2.5.0"
+CHECKREQS_DISK_BUILD="2400M"
+CHECKREQS_DISK_USR="512M"
+CHECKREQS_MEMORY="1024M"
+
+inherit check-reqs eapi7-ver eutils flag-o-matic multilib multiprocessing pax-utils python-single-r1 scons-utils systemd toolchain-funcs user
```
Comment (Zlogene) [inline dev-db/mongodb/mongodb-3.6.7.ebuild]: where is `eutils` being used?

Comment (hydrapolic) [reply dev-db/mongodb/mongodb-3.6.7.ebuild]: Seems like a relic from the past. Thanks, removed.

**Comment by ultrabug** [comment]
Thanks guys!

**Comment by hydrapolic** [comment]
Thank you @ultrabug

**Pull request closed without merging**